src/common/CustomIconButton/CustomIconButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import {
+  EuiButton,
+} from '@elastic/eui';
+
+const CustomIconButton = ({
+  iconType, href, target, label,
+}) => {
+  if (process.env.NODE_ENV === 'test') {
```

Comment:
I would suggest adding a utility to check if the env is DEV. Something like `isDevelopment` but as you don't need this too much it's fine to leave it as it is now.